config/olm/kubernetes/0.10.0/dynatrace-monitoring.v0.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,802 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "dynatrace.com/v1alpha1",
+          "kind": "OneAgent",
+          "metadata": {
+            "name": "oneagent",
+            "namespace": "dynatrace"
+          },
+          "spec": {
+            "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+            "tolerations": [
+              {
+                "effect": "NoSchedule",
+                "key": "node-role.kubernetes.io/master",
+                "operator": "Exists"
+              }
+            ],
+            "useUnprivilegedMode": true
+          }
+        },
+        {
+          "apiVersion": "dynatrace.com/v1alpha1",
+          "kind": "OneAgentAPM",
+          "metadata": {
+            "name": "oneagentapm",
+            "namespace": "dynatrace"
+          },
+          "spec": {
+            "apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api"
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Monitoring,Logging & Tracing,OpenShift Optional
```

Comment:
> OpenShift Optional
Should this be here?

Reply:
Looks weird.
We used to have it for other releases as well.
However removed in the base and for K8s and OCP file